core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -68,27 +71,26 @@ public HadoopTables(Configuration conf) {
    */
   @Override
   public Table load(String location) {
-    TableOperations ops = newTableOps(location);
```

Comment:
An alternative to this implementation that I considered was modifying findTable itself. The problem with this is that to do that implementation we need to extract the loadMetadataTable from below. I think this is a cleaner solution but we could do something where we make a StaticTables.java which findTables uses to either return a basetable or metadataTable

Reply:
I like this. It is more straightforward than the previous implementation.